tools/percolator/rollout_nested_collection.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="rollout_nested_collection" name="Rollout nested list" version="3.1">
+    <description>into a single long list</description>
```

Comment:
I haven't followed the discussion on gitter, but is this really needed and if so would this not a better tool-collection tool directly in Galaxy?